emails/password_warning.spt

```diff
@@ -0,0 +1,5 @@
+{{ _("Update your Liberapay password") }}
+
+[---] text/html
+<p>{{ _("{0}",message) }}</p>
```

Comment:
This isn't right, `_("{0}", message)` is a no-op. You shouldn't send an already-translated message to a notification template. Instead you should send the variables that the template needs to generate the message. In this case it could be either `password_strength` and `password_leak_count`, or a `password_status` string (possible values: `'weak'`, `'common'`, `'compromised'`). The latter is probably better because it allows changing the thresholds without breaking old notifications.